### PR TITLE
Holy water now slightly heals the Chaplain

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
add: Holy water now has a 45% chance to slightly heal the chaplain every cycle, in addition to causing increasing brain damage and increasing the rate at which mannitol metabolizes
/:cl:

SLIGHT heal. Really low heal. Also it causes increasing brain damage, and makes mannitol metabolize faster.

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -197,35 +197,43 @@
...
-				SSticker.mode.remove_cultist(M.mind, 1, 1)
-			else if(is_servant_of_ratvar(M))
-				remove_servant_of_ratvar(M)
-			M.jitteriness = 0
-			M.stuttering = 0
-			holder.remove_reagent(id, volume)	// maybe this is a little too perfect and a max() cap on the statuses would be better??
-			return
+	if(M.mind.assigned_role != "Chaplain")
+		if(!data) data = 1
+		data++
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: NO

STOP

USE THE FUCKING CURRENT_CYCLE VAR AAAAAAAAAAAAAAAAAAAAAAAAAAA

**Comment by CPTANT** [comment]
Why?

**Comment by Chronitonity** [comment]
You might as well just let him hit himself over the bible and skip the extra step of water if you want him to be a self healing monstrosity

**Comment by kevinz000** [comment]
What the fuck no.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -197,35 +197,43 @@
...
+			else if(is_servant_of_ratvar(M) && prob(8))
+				switch(pick("speech", "message", "emote"))
+					if("speech")
+						clockwork_say(M, "...[text2ratvar(pick("Engine... your light grows dark...", "Where are you, master?", "He lies rusting in Error...", "Purge all untruths and... and... something..."))]")
+					if("message")
+						to_chat(M, "<span class='boldwarning'>[pick("Ratvar's illumination of your mind has begun to flicker", "He lies rusting in Reebe, derelict and forgotten. And there he shall stay", \
+						"You can't save him. Nothing can save him now", "It seems that Nar-Sie will triumph after all")].</span>")
+					if("emote")
+						M.visible_message("<span class='warning'>[M] [pick("whimpers quietly", "shivers as though cold", "glances around in paranoia")].</span>")
+		if(data >= 75)	// 30 units, 135 seconds
```
Comment (Supermichael777) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: deconversion should be faster than this its disgustingly slow.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Supermichael777) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: water doesn't  even HAVE and on mob life.

also the comment down below is probably a bit dated now i think its SLOWER

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 2 above)
Comment (PraiseRatvar) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: its fast enough

**Comment by MoreRobustThanYou** [comment]
Now it causes brain damage too!

**Comment by The-clown-known-as-Pepe** [comment]
vape holy water/manitol mix

be healed passively until end of ages

**Comment by MoreRobustThanYou** [comment]
@The-clown-known-as-Pepe i just made it cause mannitol to metabolize at an ever-increasing rate

**Comment by PKPenguin321** [comment]
>smack water tank with bible
>get infinite free healing chems

🤔

**Comment by MoreRobustThanYou** [comment]
@PKPenguin321 did I mention it makes you retarded as fuck? as in, the longer it's in you, the more brain damage it causes? it also makes mannitol metabolize MUCH faster

**Comment by BeeSting12** [comment]
dude brain damage is a no, literally everyone will hate sec every cult round. bad enough it makes them drunk, this will make them hate me even more

**Comment by DoctoreMad** [comment]
@BeeSting12 It only causes the chaplain brain damage

**Comment by kevinz000** [comment]
>oh ill make this do brain damage

>>WHAT? BRAIN DAMAGE DOESNT DO ANYTHING THAT LETHAL?!!!

>>>LETS MAKE BRAIN DAMAGE LETHAL :DDDD

No.
Stop.
Just, STOP.

**Pull request closed without merging**